s-ramp-ide/pom.xml

```diff
@@ -0,0 +1,126 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>cz.muni.fi.sramp</groupId>
+	<artifactId>s-ramp-ide-parent</artifactId>
+	<!-- <version>1.0.0</version> -->
+	<packaging>pom</packaging>
+
+	<parent>
+		<groupId>org.overlord.sramp</groupId>
+		<artifactId>s-ramp</artifactId>
+		<version>0.7.0.Final</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<properties>
+		<!--<sramp.version>${project.version}</sramp.version> -->
+		<tycho.version>0.21.0</tycho.version>
+		<kepler-repo.url>http://download.eclipse.org/releases/kepler</kepler-repo.url>
+	</properties>
+
+	<modules>
+		<module>/srampRepositoryBrowser</module>
+		<module>/srampRepositoryBrowser.test</module>
+	</modules>
+
+	<repositories>
+		<repository>
+			<id>swtbot</id>
+			<url>http://download.eclipse.org/technology/swtbot/releases/2.2.1/</url>
+			<layout>p2</layout>
+		</repository>
+		<repository>
+			<id>kepler</id>
+			<url>${kepler-repo.url}</url>
+			<layout>p2</layout>
+		</repository>
+	</repositories>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-maven-plugin</artifactId>
+				<version>${tycho.version}</version>
+				<extensions>true</extensions>
+			</plugin>
+
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>target-platform-configuration</artifactId>
+				<version>${tycho.version}</version>
+				<configuration>
+					<environments>
+						<environment>
+							<os>linux</os>
+							<ws>gtk</ws>
+							<arch>x86</arch>
+						</environment>
+						<environment>
+							<os>linux</os>
+							<ws>gtk</ws>
+							<arch>x86_64</arch>
+						</environment>
+						<environment>
+							<os>win32</os>
+							<ws>win32</ws>
+							<arch>x86</arch>
+						</environment>
+						<environment>
+							<os>win32</os>
+							<ws>win32</ws>
+							<arch>x86_64</arch>
+						</environment>
+						<environment>
+							<os>macosx</os>
+							<ws>cocoa</ws>
+							<arch>x86_64</arch>
+						</environment>
+					</environments>
+				</configuration>
+			</plugin>
+		</plugins>
+		<pluginManagement>
+		    <plugins>
+		    <plugin>
+		    <groupId>org.eclipse.tycho</groupId>
+			<artifactId>tycho-p2-plugin</artifactId>
```

Comment:
This plugin uses another tycho plugin. I think it is target-platform-configuration plugin.